src/wow.coffee

```diff
@@ -152,6 +173,14 @@ class @WOW
     @applyStyle(box)
     box.className = "#{box.className} #{@config.animateClass}"
     @config.callback(box) if @config.callback?
+    @util().emitEvent(box, @wowEvent)
+
+    @util().addEvent(box, 'animationend', @resetAnimation)
+    @util().addEvent(box, 'oanimationend', @resetAnimation)
+    @util().addEvent(box, 'webkitAnimationEnd', @resetAnimation)
+    @util().addEvent(box, 'MSAnimationEnd', @resetAnimation)
```

Comment:
There is a "o" reference (line 179). From what I've read you don't need the moz prefix as mozilla followed the W3C standard pretty soon.

Some references:
http://www.sitepoint.com/css3-animation-javascript-event-handlers/
https://gist.github.com/amcintyre-cs/3258186